release/release-notes/data-prepper.release-notes-1.3.0.md

```diff
@@ -0,0 +1,60 @@
+## 2022-03-15 Version 1.3.0
+
+---
+
+### Important Deprecation
+* We have updated the pipeline definition to support `processor:` as replacement of `prepper:` which has been deprecated and will be fully removed in 2.0. ([#655](https://github.com/opensearch-project/data-prepper/pull/655), [#667](https://github.com/opensearch-project/data-prepper/pull/667))
+
+### Features
+* AggregateProcessor for generic stateful aggregation ([#839](https://github.com/opensearch-project/data-prepper/pull/839), [#850](https://github.com/opensearch-project/data-prepper/pull/850), [#931](https://github.com/opensearch-project/data-prepper/pull/931), [#969](https://github.com/opensearch-project/data-prepper/pull/969), [#1022](https://github.com/opensearch-project/data-prepper/pull/1022), [#1046](https://github.com/opensearch-project/data-prepper/pull/1046))
+* DateProcessor to extract dates from fields in events ([#971](https://github.com/opensearch-project/data-prepper/pull/971), [#1014](https://github.com/opensearch-project/data-prepper/pull/1014))
+* Processors to support mutate, alter, and delete fields from Events ([#1002](https://github.com/opensearch-project/data-prepper/pull/1002))
+* KeyValueProcessor to support parsing messages with key-value strings such as queries and properties ([#872](https://github.com/opensearch-project/data-prepper/pull/872))
+* DropProcessor to filter out (remove/drop) entire events ([#801](https://github.com/opensearch-project/data-prepper/pull/801))
+
+### Enhancements
+* Add Data Prepper expression evaluator ([#1024](https://github.com/opensearch-project/data-prepper/pull/1024), [#1027](https://github.com/opensearch-project/data-prepper/pull/1027), [#1090](https://github.com/opensearch-project/data-prepper/pull/1090), [#1065](https://github.com/opensearch-project/data-prepper/pull/1065), [#1153](https://github.com/opensearch-project/data-prepper/pull/1153), [#1155](https://github.com/opensearch-project/data-prepper/pull/1155), [#1169](https://github.com/opensearch-project/data-prepper/pull/1169), [#1177](https://github.com/opensearch-project/data-prepper/pull/1177), [#1178](https://github.com/opensearch-project/data-prepper/pull/1178))
+* Add conditional to DropEventsProcessor ([#1174](https://github.com/opensearch-project/data-prepper/pull/1174))
+* Support for Data Prepper expression in Data Prepper core ([#1157](https://github.com/opensearch-project/data-prepper/pull/1157))
```

Comment:
nit: This line is redundant with [line 16](https://github.com/opensearch-project/data-prepper/pull/1201/files#diff-46cb9eccea41269b336de54f506849fb80fe6a1a13d643bb1eb2a7f479d46309R16)